cpp/Dynamic Programming/coin_change.cpp

```diff
@@ -0,0 +1,49 @@
+// C++ program for coin change problem. 
```

Comment:
Hi, @nut556 you have already pushed this algorithm in your previous PR. Can you remove this algo from this commit?